modules/core/src/parallel.cpp

```diff
@@ -45,6 +45,10 @@
 #include <opencv2/core/utils/configuration.private.hpp>
 #include <opencv2/core/utils/trace.private.hpp>
 
+#define min_non_zero(val_1, val_2)                                                    \
+   (((val_1 > 0) && (val_2 > 0) && (val_1 > val_2)) || ((val_1 == 0) && (val_2 > 0))) \
+   ? val_2 : val_1
```

Comment:
Please implement this as `static inline` function near `getNumberOfCPUs()` (line 830)

Reply:
Sure